### DISCUSSION TITLE
r/city-china-zhejiang-zhoushan-daishan

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#25] [Announcements]
# r/city-china-zhejiang-zhoushan-daishan

游记 | 露营在海边只因订不到酒店

https://blog.ops-coffee.cn/r/city-china-zhejiang-zhoushan-daishan

**Comment by zhangxiang012** [comment]
这么热闹的暑假，这个景点你拍出来竟然没有多少游客的样子

**Reply by ops-coffee** [reply]
人多的地方基本都没咋呆，似乎就海钾公园和蓝白小镇人多，其他的还好
